src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothEnumerations.cs

```diff
@@ -1307,6 +1307,22 @@ public enum BluetoothGattRequestType
         Write = 1,
     }
 
+    /// <summary>
+    /// Enumeration for the GATT service changed type.
+    /// </summary>
+    /// <since_tizen> 9 </since_tizen>
+    public enum BluetoothGattServiceChangedType
```

Comment:
the "Changed" is in the enum name,  it would be acceptable if the description clears the meaning, but the "ServiceChangeType" would be better IMO

Reply:
Thanks. I changed to ServiceChangeType.